content/en/blog/2024/prometheus-compatibility-survey/index.md

```diff
@@ -0,0 +1,164 @@
+---
+title: Insights from the Prometheus Compatibility Survey
+linkTitle: Prometheus Compatibility Survey
+date: 2024-06-10
+author: '[David Ashpole](https://github.com/dashpole) (Google)'
+issue: https://github.com/open-telemetry/sig-end-user/issues/24
+sig: End-User SIG
+cSpell:ignore: Ashpole
+---
+
+[Prometheus](https://prometheus.io/) and OpenTelemetry are two of the most
+active and popular projects in the
+[CNCF observability landscape](https://landscape.cncf.io/guide#observability-and-analysis--observability).
+The two communities have been working together since the early days of
+OpenTelemetry to improve the compatibility between the two projects. The
+OpenTelemetry Prometheus SIG has been leading this effort, with the active
+participation of maintainers from both OpenTelemetry and Prometheus.
+
+At this point, there is a
+[detailed, experimental specification](/docs/specs/otel/compatibility/prometheus_and_openmetrics/)
+describing how to convert between the OpenTelemetry metrics format and the
+Prometheus metrics format. It has been used to implement Prometheus (pull)
```

Comment:
Add link for `prometheus metrics format`

Reply:
done